src/main/java/com/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -51,6 +53,7 @@ public ResponseEntity<Object> handleEntityNotFoundException(
         ResponseBody responseBody = new ResponseBody(LocalDateTime.now(),
                 HttpStatus.NOT_FOUND, List.of(ex.getMessage()));
         int exceptionStatus = 404;
+        log.error(ex.getMessage(), ex);
```

Comment:
client exceptions (4xx) are not considered as server errors, so it's better to not log it on error level